src/Generator.cpp

```diff
@@ -1402,7 +1419,7 @@ void GeneratorBase::pre_generate() {
     advance_phase(GenerateCalled);
     GeneratorParamInfo &pi = param_info();
     user_assert(!pi.outputs().empty()) << "Must use Output<> with generate() method.";
-    user_assert(get_target() != Target()) << "The Generator target has not been set.";
+    user_assert(get_target() != Target() || is_serializable()) << "The Generator target has not been set.";
```

Comment:
per above comments, pretty sure we want to remove the inserted `|| is_serializable()` here

Reply:
Fixed!